code/index.html

```diff
@@ -4,27 +4,243 @@
+          </video>
+        </div>
+      </div>
+    </header>
+
+    <div class="main-content">
+      <!-- Signup form -->
+
+      <section class="main-content__section">
+        <form
+          class="signup-form"
+          action="http://httpbin.org/anything"
+          method="POST"
+        >
+          <div class="input-container">
+            <label for="name">Name:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="text"
+              placeholder="Your Name"
+              id="name"
+              name="name"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="last-name">Last Name:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="text"
+              placeholder="Your Last Name"
+              id="last-name"
+              name="last-name"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="email">Enter your email:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="email"
+              placeholder="Your Email"
+              id="email"
+              name="email"
+              required
+            />
+          </div>
+
+          <div class="input-container">
+            <label for="password">Password:</label><br />
+            <input
+              class="input-container__input input-container__input--full-width"
+              type="password"
+              placeholder="Your Password"
+              id="password"
+              name="pasword"
+              required
+              maxlength="3"
+            />
+          </div>
+
+          <div class="input-container input-container--checkbox">
+            <b>Gender:</b>
+
+            <label for="male">
+              <input
+                type="radio"
+                id="male"
+                name="gender"
+                value="male"
+              />Male</label
+            ><br />
 
-    <!-- Signup form -->
+            <label for="female">
+              <input
+                type="radio"
+                id="female"
+                name="gender"
+                value="female"
+              />Female</label
+            ><br />
+
+            <label for="other"
+              ><input
+                type="radio"
+                id="other"
+                name="gender"
+                value="other"
+              />Other</label
+            >
+          </div>
+
+          <div class="input-container input-container--checkbox">
+            <b>Did you slept well?</b>
+            <br />
+
+            <label for="yes"
+              ><input type="checkbox" id="yes" name="question" value="yes" />
```

Comment:
something wrong happened here, angle bracker from `label` escaped to the next line ;)